plugins/hls-cabal-plugin/src/Ide/Plugin/Cabal.hs

```diff
@@ -82,6 +86,24 @@ instance Pretty Log where
         <+> "for cursor position:"
         <+> pretty position
     LogCompletions logs -> pretty logs
+    LogCabalAdd logs -> pretty logs
+
+-- | Some actions with cabal files originate from haskell files.
+-- This descriptor is needed to handle these cases.
+haskellInteractionDescriptor :: Recorder (WithPriority Log) -> PluginId -> PluginDescriptor IdeState
+haskellInteractionDescriptor recorder plId =
+  (defaultPluginDescriptor plId "Provides the cabal-add code action in haskell files")
```

Comment:
Change the description to be slightly more generic, this descriptor will not always only work for `cabal-add` only.